test/boost/database_test.cc

```diff
@@ -1145,5 +1150,10 @@ SEASTAR_TEST_CASE(drop_table_with_snapshots) {
         auto cf_dir_exists = co_await file_exists(cf_dir);
         BOOST_REQUIRE_EQUAL(cf_dir_exists, true);
         co_return;
-    });
+    }, db_cfg_ptr);
```

Comment:
I still don't understand why this function has two (even three?) copies of the same code?

Reply:
I think I understand now - the three copies do different things, but only the first one uses the "auto_snapshot" parameter as far as I can see, so maybe they could/should have been separate functions.